opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -183,6 +184,10 @@ configure() {
   initConfig
   applyConfd
   applyOverlayConfig
+  chmod a+x "$MINION_PROCESS_ENV_CFG"
```

Comment:
Instead of making the file executable, can we just read the value we want out of it, then export the environment variable for JAVA_OPTS to the value we read out of the file?

Also keep in mind that we are only creating the file if some config has been provided so we need a conditional to check if the file is present and only override the options in that case.

Only export the env variable if we found the config file and we were able to parse java options out of it, otherwise noop and use existing behavior.

Reply:
I changed the behavior to read the file line by line, skipping comments, and exporting `key=value` settings. I had to remove the quotes from the file, i.e. before the change the file contained

```
JAVA_OPTS="..."
```

now it contains

```
JAVA_OPTS=...
```